addons/survey/models/survey_survey.py

```diff
@@ -356,6 +357,11 @@ def _compute_scoring_type(self):
             elif not survey.scoring_type:
                 survey.scoring_type = 'no_scoring'
 
+    @api.depends('survey_type')
+    def _compute_session_available(self):
+        for survey in self:
+            survey.session_available = survey.survey_type not in {'survey', 'assessment'}
```

Comment:
Hey @flch-odoo,

I see we went for the hiding then :) just a comment here: as we also have the 'appraisal' type that does not show the live session, maybe we could instead write `in {'live_session', 'custom'}` to avoid an override in the hr_appraisal_survey.

What do you think? Cheers!

Reply:
I though it's more natural to opt out rather then opt in Live Sessions, and we need a PR to simplify the views in hr_appraisal_survey either way...
No strong opinion here ;) I'll check with François if he has some insight into other people's needs.

Reply:
Well, as it's unlikely anyone needs another type, let's go for less code. Thank you for the comment.